src/transit.js

```diff
@@ -736,6 +738,7 @@ class Transit {
 			level: ctx.level,
 			tracing: ctx.tracing,
 			parentID: ctx.parentID,
+			traceID: ctx.traceID,
```

Comment:
It's a breaking change in the protocol, so it can be merged only in 0.15 or it can't work with schema-based serializers.

Reply:
I don't need it to work with schema-based serializers at the moment. Will the serializer implementations safely ignore the new property if they don't support it? The tracing middleware handles an undefined `traceID` in the context.